data/data/aws/vpc/vpc.tf

```diff
@@ -18,6 +18,17 @@ resource "aws_vpc" "new_vpc" {
   )
 }
 
+resource "aws_vpc_endpoint" "elasticloadbalancing" {
```

Comment:
So i don't think we should be creating these endpoints in IPI as they don't provide great value to the cluster.

If they do, let's start a general discussion about these elsewhere, github issue / enhacements.